cmd/openfga/main_test.go

```diff
@@ -563,7 +562,7 @@ func GRPCCheckTest(t *testing.T, tester OpenFGATester) {
 			},
 		},
 		{
-			name: "blerb",
+			name: "union_with_direct_check_single_operand",
```

Comment:
Do we want `this_type` or `ThisType` for test names? In the end, `ThisType` is easy to search by copy and paste.

Reply:
`this_type` is as well. `this type` is bad because the whitespace gets `_` injected into it.

I'm fine with either one. We should standardize. Let's discuss between you @miparnisari and myself and we can strive to do so going forward.